docker/autoware-ai-test-packages.txt

```diff
@@ -0,0 +1,29 @@
+amathutils_lib
+autoware_build_flags
+autoware_config_msgs
+autoware_lanelet2_msgs
+autoware_lanelet2_ros_interface
+autoware_msgs
+dead_reckoner
+hardcoded_params
+jsk_recognition_msgs
+lanelet2
+lanelet2_core
+lanelet2_examples
+lanelet2_extension
+lanelet2_io
+lanelet2_maps
+lanelet2_matching
+lanelet2_projection
+lanelet2_python
+lanelet2_routing
+lanelet2_traffic_rules
+lanelet2_validation
+lidar_localizer_ros2
+map_file_ros2
+mrt_cmake_modules
+ndt_cpu
+ndt_gpu
+points_downsampler
+twist_filter
+twist_gate

```

Comment:
We should remove:
amathutils_lib
autoware_build_flags
autoware_config_msgs
autoware_lanelet2_msgs
autoware_lanelet2_ros_interface
autoware_msgs
jsk_recognition_msgs
mrt_cmake_modules
ndt_cpu
ndt_gpu